src/convert.js

```diff
@@ -136,7 +139,8 @@ function projectX(x) {
 }
 
 function projectY(y) {
-    const sin = Math.sin(y * Math.PI / 180);
+    const clampedY = y < minVectorLatitude ? minVectorLatitude : y > maxMercatorLatitude ? maxMercatorLatitude : y;
+    const sin = Math.sin(clampedY * Math.PI / 180);
     const y2 = 0.5 - 0.25 * Math.log((1 + sin) / (1 - sin)) / Math.PI;
-    return y2 < 0 ? 0 : y2 > 1 ? 1 : y2;
+    return y2;
```

Comment:
Nit: If we do the `8191 / 8192` hack, I think it should follow after the projection (not before). Currently (looking at tests) the clamp is 1px off (3094 instead of 3095). E.g. we do `return Math.max(y, 8191 / 8192)`;

Reply:
>`return Math.max(y2, 8191 / 8192)`

@mourner would this clamp features on _every_ tile locally to 8191?

Reply:
@peterqliu no, since this is a global projection — it projects latitude into 0..1 range (0 is near the north pole, 1 is near the south one). Tile-aware projection happens further down the line